workflow-ui/core-android/src/main/java/com/squareup/workflow1/ui/BackPressHandler.kt

```diff
@@ -24,7 +25,9 @@ public typealias BackPressHandler = () -> Unit
  * That means that this is a last-registered-first-served mechanism, and that it is
  * compatible with Compose back button handling.
  */
+@Suppress("DEPRECATION")
 @WorkflowUiExperimentalApi
+@Deprecated("Use setOrClearBackHandler")
```

Comment:
```suggestion
@Deprecated("Use setBackHandler")
```

I see you went through some naming iterations 😄